mssqlcli/mssqlclioptionsparser.py

```diff
@@ -11,143 +11,143 @@
 MSSQL_CLI_ROW_LIMIT = u'MSSQL_CLI_ROW_LIMIT'
 MSSQL_CLI_RC = u'MSSQL_CLI_RC'
 
+from mssqlcli.i18n import translation
```

Comment:
Let's undo this changes to this file since we're only localizing the 'goodbye!' message.